client-v2/src/services/__tests__/faciaApi.spec.ts

```diff
@@ -98,6 +99,38 @@ describe('faciaApi', () => {
       await getArticlesBatched(articleIds);
     });
   });
+  describe('getTagOrSectionTitle', () => {
+    it('should return the title of the tag or section from a given CAPI response, if it exists', () => {
+      expect(
+        getTagOrSectionTitle({
+          response: {
+            status: 'ok',
+            results: [],
+            tag: {
+              id: 'tagId',
+              type: 'tag',
+              webUrl: 'exampleUrl',
+              webTitle: 'Example tag title'
+            }
+          }
+        })
+      ).toEqual('Example tag title');
+      expect(
+        getTagOrSectionTitle({
+          response: {
+            status: 'ok',
+            results: [],
+            tag: {
```

Comment:
I thiiiink this is meant to be a `section` key but I may be wrong?

Reply:
Hopefully I'm not wrong because that means I understand the rest of it and will approve it accordingly!

Reply:
hah. ohh boy. Yes, yes it is. Thanks Rich! 👍 the system wooorks